data/privacy.md

```diff
@@ -0,0 +1,49 @@
+### What personal data do we collect?
+
+We may collect personal data about you, directly from you, when you sign in on our website or when you purchase a ticket for our events. This type of information may include:
+- name
+- e-mail address
+- user name
+- and/or other content you submit to our website or ticketing platform and any other information you provide voluntarily to us
+
+We may use cookies to associate accounts and/or devices with you.
+
+### For which purposes do we use your personal data?
+
+We collect your personal data to provide you with required services or information. The following are examples of how we may use personal data we collect:
+
+- Provide you with services that you request from us - receiving our newsletters, event-related e-mails and confirmation for your participation fee;
+- Communicate regularly with you to inform you about event updates, content, and special ticket offers that may be of interest to you;
+- Contact you if this is required to inform you of any event changes;
+- To inform you about changes in the agenda;
+- Send you an invoice for your ticket/participation fee;
+- Resolve any issues that might arise during your event registration;
+- Anticipate and resolve problems with the tickets supplied to you;
+- Improve your user experience on our websites;
+- Participate in our communities, including our social media channel/pages and blogs;
+- Store your preferences for future interactions and communications from us;
+- Share your feedback about our events and conferences;
+- Create external marketing & analytics reports;
+- Prevent illegal activities, suspected fraud and potential threats to our network;
+- Analyse the characteristics of our participants at an aggregated level and participant utilization of our events in order to understand participants and their needs, to improve our events, to design new events and for further event and community development.
```

Comment:
Should we call out anonymity here?

Reply:
👍 done, and simplified this sentence a bit